imblearn/over_sampling/_smote.py

```diff
@@ -766,6 +774,9 @@ class SMOTENC(SMOTE):
     --------
     SMOTE : Over-sample using SMOTE.
 
+    SMOTEN : Over-sample using the SMOTE variable specifically for categorical
```

Comment:
SMOTEN : Over-sample using the SMOTE variant specifically for nominal features only.